container-storage-setup.sh

```diff
@@ -721,18 +719,21 @@ scan_disks() {
 }
 
 create_partition_sfdisk(){
-  local dev="$1" size
+  local dev="$1" size output
   # Use a single partition of a whole device
   # TODO:
   #   * Consider gpt, or unpartitioned volumes
   #   * Error handling when partition(s) already exist
   #   * Deal with loop/nbd device names. See growpart code
   size=$(( $( awk "\$4 ~ /"$( basename $dev )"/ { print \$3 }" /proc/partitions ) * 2 - 2048 ))
-    cat <<EOF | sfdisk $dev
+  if ! output=$(cat <<EOF | sfdisk $dev
```

Comment:
we can get rid of capturing output in a variable and supressing it once you stop returning partition name from create_partition()